data/examples/vm/values-istio-meshexpansion-gateways.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: install.istio.io/v1alpha2
+kind: IstioControlPlane
+spec:
+  values:
+    global:
+      # Provides dns resolution for global services
+      podDNSSearchNamespaces:
+        - global
+        - "{{ valueOrDefault .DeploymentMeta.Namespace \"default\" }}.global"
```

Comment:
how does this template part works?

Reply:
Just copy from https://github.com/istio/operator/blob/master/data/examples/multicluster/values-istio-multicluster-gateways.yaml and verified this works for vm mesh expansion scenario.

Reply:
This will be added to sidecar injection template and rendered by the sider injector, see: https://github.com/istio/installer/blob/master/istio-control/istio-autoinject/files/injection-template.yaml#L390-L392

Reply:
I have removed the podDNSSearchNamespaces section and verified it works